src/ngraph/frontend/onnx_import/op/lp_norm.cpp

```diff
@@ -37,22 +39,33 @@ namespace ngraph
                 NodeVector lp_norm(const Node& node)
                 {
                     const std::shared_ptr<ngraph::Node> data{node.get_ng_inputs().at(0)};
-                    std::int64_t axis{node.get_attribute_value<std::int64_t>("axis", -1)};
                     const std::int64_t p_norm{node.get_attribute_value<std::int64_t>("p", 2)};
-                    std::size_t valid_axis =
+
+                    const std::int64_t axis{node.get_attribute_value<std::int64_t>("axis", -1)};
+                    const std::size_t valid_axis =
                         common::validate_axis(node, axis, data->get_shape().size());
 
                     ASSERT_VALID_ARGUMENT(node, p_norm == 1 || p_norm == 2)
                         << "Invalid `p` attribute value: " << p_norm
                         << "Only normalization of 1st or 2nd order is supported.";
 
-                    const AxisSet reduction_axes{valid_axis};
                     std::shared_ptr<ngraph::Node> norm = ngraph::builder::lp_norm(
-                        data, reduction_axes, static_cast<std::size_t>(p_norm));
-                    norm = std::make_shared<ngraph::opset0::Broadcast>(
-                        norm, data->get_shape(), reduction_axes);
+                        data, AxisSet{valid_axis}, static_cast<std::size_t>(p_norm));
+
+                    const auto target_shape = default_opset::Constant::create(
+                        element::i64, Shape{data->get_shape().size()}, data->get_shape());
+
+                    std::vector<size_t> axes_values(data->get_shape().size());
```

Comment:
Please add a line comment explaining what these 3 lines are doing and why they are needed.